exercises/practice/darts/.approaches/if-statements/content.md

```diff
@@ -0,0 +1,45 @@
+# `if`  statements
+
+```java
+import java.lang.Math;
+
+class Darts {
+
+    int score(double x, double y) {
+        var radius = (Math.sqrt((x * x) + (y * y)));
```

Comment:
Just in case you didn't know, I _think_ you could also use:

```suggestion
        var radius = Math.hypot(x, y);
```

But your version is more readable I think 🤷

Reply:
Yes, I saw `hypot` used in a student request for mentoring. I failed geometry in high school, so I just stayed with `sqrt`. 😄

Reply:
Hah, same here! I was awful at geometry (still am), and saw `hypot` used by a student in the Kotlin track. :D